src/lib/common/include/sol-pin-mux-modules.h

```diff
@@ -32,6 +32,11 @@
 
```

Comment:
But that is intended for the next release, or we should bump it in a separated commit? I think this way will be very easy to forget to bump it.